CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.8] - 2020-04-28
+### 🚀 Features
+- Adding resource for toybox to designate and undesignate toy for return.
+- Adding resource for products get products with query params.
+
```

Comment:
Please add this under 'unreleased' as it's not released yet. When merging from develop to master, we add a release tag and date

```suggestion
## [Unreleased]
### 🚀 Features
- Adding resource for toybox to designate and undesignate toy for return.
- Adding resource for products get products with query params.

```